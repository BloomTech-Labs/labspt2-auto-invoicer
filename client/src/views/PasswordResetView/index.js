import React from "react";

class PasswordResetView extends React.Component {
  render() {
    return <section>Password Reset view</section>;
  }
}

export default PasswordResetView;
