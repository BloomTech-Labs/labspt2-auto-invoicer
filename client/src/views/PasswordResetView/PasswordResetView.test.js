import React from "react";
import { render } from "react-testing-library";
import "jest-dom/extend-expect";
import PasswordResetView from "../PasswordResetView";

describe("<PasswordResetView />", () => {
  it("renders Password Reset view", () => {
    const { asFragment } = render(<PasswordResetView />);
    expect(asFragment()).toMatchSnapshot();
  });
});
