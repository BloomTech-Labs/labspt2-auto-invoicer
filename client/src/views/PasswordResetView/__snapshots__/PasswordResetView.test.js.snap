// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PasswordResetView /> renders Password Reset view 1`] = `
<DocumentFragment>
  <section>
    Password Reset view
  </section>
</DocumentFragment>
`;
